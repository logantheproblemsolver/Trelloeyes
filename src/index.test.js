import React from 'react';
import ReactDOM from 'react-dom';
import Card from "./Card";
import List from "./List";
import renderer from 'react-test-renderer';



it('card renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title={'Test Title'} content={'Test Content'}/>, div);
    ReactDOM.unmountComponentAtNode(div);
});




it('list renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List cards={[]} />, div);
    ReactDOM.unmountComponentAtNode(div);
});


it('renders the UI as expected', () => {
    const tree = renderer 
        .create(<Card title={'Test Title'} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders the UI as expected', () => {
    const tree = renderer 
        .create(<List cards={[]} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})