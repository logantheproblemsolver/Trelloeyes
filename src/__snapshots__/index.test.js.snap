// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Test Title
  </h3>
  <p />
</div>
`;

exports[`renders the UI as expected 2`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2 />
  </header>
  <div
    className="List-cards"
  >
    <button
      className="List-add-button"
      type="button"
    >
      + Add Random Card
    </button>
  </div>
</section>
`;
